tox.ini

```diff
@@ -10,6 +10,7 @@ python =
     3.9: py39
     3.10: py310
     pypy-3.6: pypy3
+	pypy-3.7: pypy3
```

Comment:
Indent?

Reply:
Oops, edited in github, it used wrong indent but looks the same...